src/tests/banners/bannerSelection.tsx

```diff
@@ -73,16 +74,44 @@ const audienceMatches = (showSupportMessaging: boolean, testAudience: BannerAudi
     }
 };
 
+const getForcedVariant = (
+    forceTestVariant: TestVariant,
+    tests: BannerTest[],
+    baseUrl: string,
+): BannerTestSelection | null => {
+    const test = tests.find(
+        test => test.name.toLowerCase() === forceTestVariant.testName.toLowerCase(),
+    );
+    const variant = test?.variants.find(
+        v => v.name.toLowerCase() === forceTestVariant.variantName.toLowerCase(),
+    );
+
+    if (test && variant) {
+        return {
+            test,
+            variant,
+            moduleUrl: `${baseUrl}/${variant.modulePath}`,
+            moduleName: variant.moduleName,
+        };
+    }
+    return null;
+};
+
 export const selectBannerTest = async (
     targeting: BannerTargeting,
     pageTracking: BannerPageTracking,
     baseUrl: string,
     getTests: () => Promise<BannerTest[]>,
     bannerDeployCaches: BannerDeployCaches,
+    forceTestVariant?: TestVariant,
```

Comment:
Tiny point - should this be called `forcedTestVariant`? (i.e. force -> forced)

Reply:
good point 👍